src/layouts/dashboard/index.js

```diff
@@ -113,66 +114,135 @@ function DashBoardOriginal() {
+//               <MiniStatisticsCard
+//                 title={{ text: "sales" }}
+//                 count="$103,430"
+//                 percentage={{ color: "success", text: "+5%" }}
+//                 icon={{
+//                   color: "info",
+//                   component: "shopping_cart",
+//                 }}
+//               />
+//             </Grid>
+//           </Grid>
+//         </SoftBox>
+//         <SoftBox mb={3}>
+//           <Grid container spacing={3}>
+//             <Grid item xs={12} lg={7}>
+//               <BuildByDevelopers />
+//             </Grid>
+//             <Grid item xs={12} lg={5}>
+//               <WorkWithTheRockets />
+//             </Grid>
+//           </Grid>
+//         </SoftBox>
+//         <SoftBox mb={3}>
+//           <Grid container spacing={3}>
+//             <Grid item xs={12} lg={5}>
+//               <ReportsBarChart
+//                 title="active users"
+//                 description={
+//                   <>
+//                     (<strong>+23%</strong>) than last week
+//                   </>
+//                 }
+//                 chart={chart}
+//                 items={items}
+//               />
+//             </Grid>
+//             <Grid item xs={12} lg={7}>
+//               <GradientLineChart
+//                 title="Sales Overview"
+//                 description={
+//                   <SoftBox display="flex" alignItems="center">
+//                     <SoftBox
+//                       fontSize={size.lg}
+//                       color="success"
+//                       mb={0.3}
+//                       mr={0.5}
+//                       lineHeight={0}
+//                     >
+//                       <Icon className="font-bold">arrow_upward</Icon>
+//                     </SoftBox>
+//                     <SoftTypography
+//                       variant="button"
+//                       color="text"
+//                       fontWeight="medium"
+//                     >
+//                       4% more{" "}
+//                       <SoftTypography
+//                         variant="button"
+//                         color="text"
+//                         fontWeight="regular"
+//                       >
+//                         in 2021
+//                       </SoftTypography>
+//                     </SoftTypography>
+//                   </SoftBox>
+//                 }
+//                 height="20.25rem"
+//                 chart={gradientLineChartData}
+//               />
+//             </Grid>
+//           </Grid>
+//         </SoftBox>
+//         <Grid container spacing={3}>
+//           <Grid item xs={12} md={6} lg={8}>
+//             <Projects />
+//           </Grid>
+//           <Grid item xs={12} md={6} lg={4}>
+//             <OrderOverview />
+//           </Grid>
+//         </Grid>
+//       </SoftBox>
+//       <Footer />
+//     </DashboardLayout>
+//   );
+// }
+Footer />
```

Comment:
the code backup comment has parts that are not commented, which is not recognized.